src/main/webapp/i18n/de/textExercise.json

```diff
@@ -6,6 +6,8 @@
                 "createLabel": "Neue Textaufgabe erstellen",
                 "editLabel": "Textaufgabe bearbeiten"
             },
+            "wordCount": "Anzahl der Wörter: {{count}}",
+            "characterCount": "Anzahl der Zeichen: {{count}}",
```

Comment:
```suggestion
            "characterCount": "Anzahl der Zeichen: {{ count }}",
```

Reply:
Done in 1f009e2b14d30346d16cfb3ed30d68418a4ee8b1